apis/server/image_bridge.go

```diff
@@ -53,6 +53,17 @@ func (s *Server) listImages(ctx context.Context, resp http.ResponseWriter, req *
 	return json.NewEncoder(resp).Encode(imageList)
 }
 
+func (s *Server) getImage(ctx context.Context, resp http.ResponseWriter, req *http.Request) error {
+	idOrRef := mux.Vars(req)["name"]
+
+	image, err := s.ImageMgr.GetImage(ctx, idOrRef)
+	if err != nil {
+		logrus.Errorf("failed to get image in containerd: %v", err)
+		return err
+	}
```

Comment:
Could you add status code 200 here explicitly?
Although by default it is 200.

Reply:
Updated